engine/src/test/scala/cromwell/WorkflowOutputsSpec.scala

```diff
@@ -63,7 +63,7 @@ class WorkflowOutputsSpec extends CromwellTestkitSpec {
     "Allow explicitly named inputs in the output section" ignore {
```

Comment:
This test needs to be discussed. It verifies that one can declare a task input as a workflow output, which is not possible in 0.20, because we don't store task inputs anymore (we didn't need to until now).
@kcibul Question: is this a required feature ? If yes it deserves its own ticket IMO.